themes/default/content/blog/introducing-the-puluminaries/index.md

```diff
@@ -10,7 +10,7 @@ date: 2022-05-03T10:04:47-05:00
 # Use the meta_desc property to provide a brief summary (one or two sentences)
 # of the content of the post, which is useful for targeting search results or social-media
 # previews. This field is required or the build will fail the linter test.
-meta_desc: We are very excited to announce and showcase our new program of community champions, the Puluminaries! 
+meta_desc: We are very excited to announce and showcase our new program of community champions, the Puluminaries!
```

Comment:
that shouldn't be the meta description which is saying its an announcement

Reply:
This file is the existing blog post - the only change was in the redirect.

Reply:
ah, got it